ui/events/flow.ts

```diff
@@ -78,6 +80,8 @@ export function setFlowMode(enabled: boolean) {
         hideNavBar();
     } else {
         screen.classList.remove('flow');
+        selectedTabArtboards.classList.add('current');
+        showNavbar.classList.add('on');
```

Comment:
please extract your codes to `tab.ts` as `showNavBar()`, just like `hideNavBar()`